src/Automation/Automation/Cmdlet/GetAzureAutomationModule.cs

```diff
@@ -43,21 +53,36 @@ protected override void AutomationProcessRecord()
             IEnumerable<Module> ret = null;
             if (!string.IsNullOrEmpty(this.Name))
             {
-                ret = new List<Module>
+                if (string.IsNullOrWhiteSpace(RuntimeVersion) || RuntimeVersion.Equals(Constants.RuntimeVersion.PowerShell51))
```

Comment:
This conditional statement appear many times. Could you extract it as a funtion?

Reply:
Updated , Exported as function.